webapp/tests/test_metrics.py

```diff
@@ -108,7 +108,42 @@ def test_find_view(self):
         #
         response = self.client.post(url, {})
         self.assertEqual(response.status_code, 400)
-        self.assertEqual(response.content, b"Missing required parameter 'query'")
+        self.assertEqual(response.content, b"Bad Request: Missing required parameter 'query'")
+
+        #
+        # invalid from/until params
+        #
+        response = self.client.post(url, {
+            'query': '*',
+            'from': 'now-1h',
+            'until': 'now-2h',
+        })
+        self.assertEqual(response.status_code, 400)
+        # response contains timestamps such as:
+        # Bad Request: Failed to instantiate find query: Invalid interval start=1582801589 end=1582797989
+        self.assertRegex(response.content, b"^Bad Request: Failed to instantiate find query: Invalid interval start=[0-9]+ end=[0-9]+$")
+
+        #
+        # Wrong type for param 'wildcards'
+        #
+        response = self.client.post(url, {
+            'query': '*',
+            'wildcards': '123a',
+        })
+        self.assertEqual(response.status_code, 400)
+        # the output in Python 2/3 slightly varies because repr() shows unicode strings differently, that's why the "u?"
+        self.assertRegex(response.content, b"^Bad Request: Invalid int value u?'123a' for param wildcards: invalid literal for int\(\) with base 10: u?'123a'$")
```

Comment:
The backslashes in `b"...literal for int\(\) with base..."` should be escaped (doubled).

(Another thing DeepSource is complaining about, which flake8 would also typically complain about.)

Reply:
You may want to rebase on latest master and resolve any new flake8 warnings. (It will probably just be the backslashes on this line.)

Reply:
Thx! done